sites/modules/@apostrophecms-pro/palette/lib/configs/2type.js

```diff
@@ -1,9 +1,9 @@
 import choices from '../choices.js';
 
-const h1 = baseProperties('heading1', 'Heading 1', '.heading-1');
-const h2 = baseProperties('heading2', 'Heading 2', '.heading-2');
-const h3 = baseProperties('heading3', 'Heading 3', '.heading-3');
-const p = baseProperties('body', 'Body Copy', '.body-copy');
+const h1 = baseProperties('heading1', 'Heading 1', '[data-rich-text] h1, [contenteditable] h1', 48);
```

Comment:
This rule scopes to rich text in both logged-in and logged-out scenarios without asking tiptap to apply classes as a workaround

Reply:
Also brought in default font sizes computed from the original typography styles